src/components/Proposal/ProposalContainer.tsx

```diff
@@ -76,6 +76,8 @@ class ProposalContainer extends React.Component<IProps, null> {
       const totalReputation = proposal.state == ProposalStates.Executed ? proposal.reputationWhenExecuted : dao.reputationCount;
       const yesPercentage = totalReputation ? Math.round(proposal.votesYes / totalReputation * 100) : 0;
       const noPercentage = totalReputation ? Math.round(proposal.votesNo / totalReputation * 100) : 0;
+      const passedByDecision = totalReputation ? proposal.votesYes / totalReputation > 0.5 : false;
```

Comment:
can totalReputation  be == 0?

Reply:
I'd be more comfortable with parens:

```
totalReputation ? (proposal.votesYes / totalReputation) > 0.5
```

Reply:
i guess it shouldn't ever be?

Reply:
oh wait, of course it can be, when the proposal is first created

Reply:
> oh wait, of course it can be, when the proposal is first created

Then will you get a divide-by-zero exception?